deployments/intersmash-deployments-shared/intersmash-deployments-shared-wildfly/pom.xml

```diff
@@ -141,7 +141,31 @@
 
     <profiles>
         <profile>
-            <id>deployments.eap</id>
+            <id>wildfly-deployments-build.eap</id>
+            <properties>
+                <!-- WildFly/EAP 8 Maven Plugin coordinates -->
+                <wildfly-maven-plugin.groupId>org.jboss.eap.plugins</wildfly-maven-plugin.groupId>
+                <wildfly-maven-plugin.artifactId>eap-maven-plugin</wildfly-maven-plugin.artifactId>
+                <wildfly-maven-plugin.version>1.0.0.Beta6-redhat-00001</wildfly-maven-plugin.version>
```

Comment:
@fabiobrz JFD has used version `1.0.0.Final-redhat-00004` for the SAML feature pack documentation, even though it's not yet available in maven central

Reply:
I chose to keep everything EAP 8 related to the latest released version, i.e. EAP 8 Beta.
This way we can be sure the CI checks that are run in the OpenShift CI infra will find the required bits.
So, we'll update everything to align with EAP 8 when it will be GA.